### PR TITLE
✨ Login page and some styling features

### PR DESCRIPTION
**Pull request by Niklas-Schaefers**

### CODE REVIEW

**Review by Dev-dfm — APPROVED** [review]
lgtm 🚀

File: src/app/components/HeaderBar/HeaderBar.module.css [hunk 1]
```diff
@@ -11,6 +11,22 @@
...
+  50% {
+    transform: rotate(-30deg);
+  }
+  75% {
+    transform: rotate(5deg);
+  }
+  100% {
+    transform: rotate(0deg);
+  }
 }
```
Comment (lmachens) [inline src/app/components/HeaderBar/HeaderBar.module.css:30]: You have the same animation multiple times in your app. Consider outsourcing it or adding it to your globals.
https://github.com/css-modules/css-modules#composing-from-other-files

File: src/app/components/Icons/LoginLogo.tsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+function LoginLogo(props: React.SVGProps<SVGSVGElement>): JSX.Element {
+  return (
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      viewBox="0 0 70.27 76.59"
+      {...props}
+    >
+      <path
+        id="Intersection_1"
+        data-name="Intersection 1"
```
Comment (lmachens) [inline src/app/components/Icons/LoginLogo.tsx]:
```suggestion
```

File: src/app/components/Labled/LabeledInput.tsx [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+type LabeledInputProps = {
+  label: string;
+  placeholder: string;
+  value: string | number;
+  type?: string;
+  required?: boolean;
+  onChange: (value: string) => void;
+};
+
+export default function LabeledInput({
```
Comment (lmachens) [inline src/app/components/Labled/LabeledInput.tsx]: What's the difference to `src/app/components/LabeledInput/LabeledInput`? Looks like a duplicate.

**Review by lmachens — COMMENTED** [review]
gj

**Review by dhnazarenko — COMMENTED** [review]
LGTM 🚀

**Pull request merged**